news/provision-front_end.sh

```
@@ -0,0 +1,55 @@
+    shift # past value
+    ;;
+esac
+done
+
+echo "Provisioning docker image $DOCKER_IMAGE"
+
+# cleanup previous deployment
+docker stop front_end || true
+docker rm front_end || true
+
+eval $(aws ecr get-login --region $AWS_DEFAULT_REGION --no-include-email)
```

Shellcheck (0.10.0)

[warning] Quote this to prevent word splitting.

(SC2046)